clientd3d/dialog.h

```diff
@@ -83,7 +84,7 @@ typedef struct {
 
 M59EXPORT void SetDescParams(HWND hParent, int flags);
 M59EXPORT void DisplayDescription(object_node *obj, BYTE flags, char *description, 
-                                  char* fixed_string, char *url);
+                                  char* fixed_string, char *url, int rarity = ITEM_RARITY_GRADE_NORMAL);
```

Comment:
We want to avoid proliferating default values; they are kind of a bad practice.  The one for ExtractObject is a convenience because there are so many callers, and it would be a pain to change them all.  But there are only a few callers of this, so no need for a default value.

Reply:
Understood.  Adjusted all callers and removed this default parameter.